ui/scss/component/_card.scss

```diff
@@ -31,6 +32,7 @@ $padding-card-horizontal: $spacing-vertical * 2/3;
   padding: 0 $padding-card-horizontal;
   margin-top: $spacing-vertical * 1/3;
   margin-bottom: $spacing-vertical * 1/3;
+  font-size: 80%
```

Comment:
I could possibly favor a reduction in card size, but I think as-is this font up being too small. Currently, the app is sort of inconsistently following material design guidelines (but trending towards consistency!). You can find these at [material.io](https://material.io).